core/domain/skill_domain_test.py

```diff
@@ -790,15 +790,33 @@ def setUp(self):
             'worked_examples_count': 1,
             'topic_name': 'topic1',
             'classroom_name': 'math',
-            'skill_model_created_on': time_in_millisecs,
-            'skill_model_last_updated': time_in_millisecs
+            'skill_model_created_on': self.time_in_millisecs,
+            'skill_model_last_updated': self.time_in_millisecs
         }
 
         self.augmented_skill_summary = skill_domain.AugmentedSkillSummary(
             'skill_id', 'description', 'en', 1, 1, 1,
             'topic1', 'math', current_time, current_time)
 
-    def test_skill_summary_gets_created(self):
+    def test_augmented_skill_summary_gets_created(self):
         self.assertEqual(
             self.augmented_skill_summary.to_dict(),
             self.augmented_skill_summary_dict)
+
+    def test_augmented_skill_summary_to_dict(self):
```

Comment:
> Sorry I misunderstood. I though this was supposed to be the only thing,

Oh, I see the issue. Your previous test name is incorrect. It would have sufficed, if the name was correct. Also, the expected dict should actually go in the test itself, not in the setUp().

Feel free to get rid of this new method once you've updated that. The way you name and frame the tests is quite important.

Reply:
Done